### PR TITLE
WA-1608: verify BSB service

### PR DESCRIPTION
**Pull request by hailefrollo**

### CODE REVIEW

File: FrolloSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7454,6 +7468,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```
Comment (sincychacko-frollo) [inline FrolloSDK.xcodeproj/project.pbxproj:7471]: Discard this?

File: FrolloSDK.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7533,6 +7548,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```
Comment (sincychacko-frollo) [inline FrolloSDK.xcodeproj/project.pbxproj:7551]: This too?

File: Sources/FrolloSDK/Payments/Payments.swift [hunk 3]
```diff
@@ -43,14 +43,15 @@ public class Payments: ResponseHandler {
          - accountNumber: Account number of the payee
          - amount: Amount of the payment
          - bsb: BSB of payee's bank
+         - isNPPAllowed: if this BSB account allow NPP payment
```
Comment (sincychacko-frollo) [inline Sources/FrolloSDK/Payments/Payments.swift:46]: Remove this too?

File: Sources/FrolloSDK/Payments/Payments.swift [hunk 4]
```diff
@@ -43,14 +43,15 @@ public class Payments: ResponseHandler {
...
          - description: desctiption of the payment (Optional)
          - paymentDate: Date of the payment (Optional)
          - reference: reference of the payment (Optional)
          - sourceAccountID: Account ID of the payment source account
          - securityCode: Verification code/ OTP for payment
          - completion: Optional completion handler with `APIPayAnyoneResponse` result if succeeds and error if the request fails
      */
-    public func payAnyone(accountHolder: String, accountNumber: String, amount: Decimal, bsb: String, description: String? = nil, paymentDate: Date? = nil, reference: String? = nil, sourceAccountID: Int64, securityCode: String? = nil, completion: @escaping (Result<PayAnyoneResponse, Error>) -> Void) {
+    public func payAnyone(accountHolder: String, accountNumber: String, amount: Decimal, bsb: String, isNPPAllowed: Bool, description: String? = nil, paymentDate: Date? = nil, reference: String? = nil, sourceAccountID: Int64, securityCode: String? = nil, completion: @escaping (Result<PayAnyoneResponse, Error>) -> Void) {
```
Comment (sincychacko-frollo) [inline Sources/FrolloSDK/Payments/Payments.swift:54]: As per our discussion, we don't need this?

Comment (hailefrollo) [reply Sources/FrolloSDK/Payments/Payments.swift:54]: I think I should keep this as future we can use for overrideMethod, am I right?

File: Sources/FrolloSDK/Payments/Payments.swift (hunk 3 above)
Comment (hailefrollo) [reply Sources/FrolloSDK/Payments/Payments.swift:46]: I think I should keep this as future we can use for overrideMethod, am I right?

File: Sources/FrolloSDK/Payments/Payments.swift (hunk 4 above)
Comment (sincychacko-frollo) [reply Sources/FrolloSDK/Payments/Payments.swift:54]: Don't think we need this, as the API is not expecting it.

**Review by dipeshfrollo — APPROVED** [review]

**Review by sincychacko-frollo — APPROVED** [review]

**Pull request merged**